tap_jira/http.py

```diff
@@ -178,10 +180,14 @@ def __init__(self, config):
             self.oauth_client_id = config.get('oauth_client_id')
             self.oauth_client_secret = config.get('oauth_client_secret')
 
+            if dev_mode and not self.access_token:
+                raise Exception("Access token config property is missing")
+
             # Only appears to be needed once for any 6 hour period. If
             # running the tap for more than 6 hours is needed this will
             # likely need to be more complicated.
```

Comment:
I'm assuming this change is targeting the 3LO flow for Jira cloud [in these docs](https://developer.atlassian.com/cloud/jira/software/oauth-2-3lo-apps/#faq1), if that's the case, I think that this comment is now out of date and should be removed.

Reply:
Also, if that's the case, shouldn't there be a timer on the refresh or a check on how long the Access Token is valid for? (if that's provided by the API)?

Reply:
@dmosorast are you talking about [this code](https://github.com/singer-io/tap-jira/blob/master/tap_jira/http.py#L272) ?
If so, it is already implemented and it is according to the API's limit of 3600 seconds.

Reply:
So, I may have to just remove this comment as it is outdated, right ?

Reply:
Ah, yeah that seems fine. I didn't realize we went the complicated route with this tap using the arbitrary thread interval :/ Not ideal, but seems fine to keep.